terraform/examples/io500/config/config.sh

```diff
@@ -30,13 +30,28 @@ ID=""
 PREEMPTIBLE_INSTANCES="true"
 SSH_USER="daos-user"
 DAOS_ALLOW_INSECURE="false"
+DAOS_POOLS='[
+  {
+    label = "io500_pool"
+    size = "1TB"
+    tier_ratio = 3
+    properties = "reclaim:disabled"
+    containers = [
+      {
+        label = "io500_cont",
+        type = "POSIX",
+        properties = "rf:0"
+      }
+    ]
+  }
+]'
```

Comment:
same leave for 83.

Reply:
Ok, I will move this changes to other PR.